postgres_composite_types/forms.py

```diff
@@ -218,6 +218,12 @@ def value_from_datadict(self, data, files, name):
             for subname, widget in self.widgets.items()
         }
 
+    def value_omitted_from_data(self, data, files, name):
+        for key in data:
+            if key.startswith('{}-'.format(name)):
```

Comment:
Calculate the comparison name outside the loop and store in a variable.

Reply:
It has be rewritten in a new commit.